app/controllers/CategorisationResultController.scala

```diff
@@ -18,35 +18,28 @@ package controllers
 
 import controllers.actions._
 import models.Scenario
-
-import javax.inject.Inject
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
-import queries.RecategorisingQuery
-import repositories.SessionRepository
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
 import views.html.CategorisationResultView
 
-import scala.concurrent.{ExecutionContext, Future}
+import javax.inject.Inject
+import scala.annotation.unused
+import scala.concurrent.ExecutionContext
 
 class CategorisationResultController @Inject() (
   override val messagesApi: MessagesApi,
   identify: IdentifierAction,
   getData: DataRetrievalAction,
   requireData: DataRequiredAction,
   val controllerComponents: MessagesControllerComponents,
-  view: CategorisationResultView,
-  sessionRepository: SessionRepository
-)(implicit ec: ExecutionContext)
+  view: CategorisationResultView
+)(implicit @unused ec: ExecutionContext)
```

Comment:
why do we need it if it's unused?

Reply:
So yeah on this one that is right we can remove it

On some of the others, one of the functions is using it so its showing as unused in the class for some reason, so those ones need to be flagged unused to keep the compiler happy :)